### ISSUE TITLE
[easy] PEP8 code style cleanup

### ISSUE THREAD

**Issue by JessicaMulein** [jseutter/ofxparse#70]
This is totally a pedantry issue and not a bug issue, but the flake8 report on this is not so great. Would to clean it up so that it is easier to find bugs (eg only printout is the bug).

**Comment by jseutter** [comment]
Yes, it would be nice to have ofxparse be PEP8 clean.  If anyone reading this has time to clean up the multitude of violations, let me know and I'll do my best to get your changes merged in a timely manner.
